format/index.md

```diff
@@ -745,21 +736,6 @@ Content-Type: application/vnd.api+json
 }
 ```
 
-If the above article's author is missing, then a `GET` request to that related
-resource would return:
-
-```http
-HTTP/1.1 200 OK
-Content-Type: application/vnd.api+json
-
-{
-  "links": {
-    "self": "http://example.com/articles/1/author"
-  },
-  "data": null
-}
-```
-
```

Comment:
I believe that the normative mention of the possible `null` response here is appropriate. It is not redundant with the "fetching relationships" section, since this section covers "fetching resources". These sections clarify that `null` is an appropriate response at a related resource URL (e.g. `/articles/1/author`).